type.proto

```diff
@@ -14,7 +14,8 @@ enum LabelType {
   LABEL_CYCLIST = 3;
   LABEL_MISC = 4;
   LABEL_GROUND = 5;
-  LABEL_MAX = 6;
+  LABEL_GATE = 6;
+  LABEL_MAX = 7;
 }
 
 enum ZoneType {
```

Comment:
I'm not sure if I have to add `ZONE_TYPE_GATE` to this enum. Any thoughts?

Reply:
what is difference `LABEL_GATE` and `ZONE_TYPE_GATE` ?

Reply:
It needs some more background information, so let me explain it. I recently added a zone type called `GateZone`. This has to be set tightly around the gate, and objects fully inside the gate are classified as `Gate`. `ZONE_TYPE_GATE` might be required if we want to create such a zone through REST API.

Reply:
ok, if the zone type is used in all product,  then we need to add it.

Reply:
@minhup I'm pretty sure gate zone is for all products, am I correct?

Reply:
@KangJH No, it's for only sensr S for now.